templates/inspec/tests/integration/build/gcp-mm.tf

```diff
@@ -838,3 +838,21 @@ resource "google_compute_node_group" "inspec-node-group" {
   size = var.node_group["size"]
   node_template = "${google_compute_node_template.inspec-template.self_link}"
 }
+
+resource "google_spanner_instance" "spanner_instance" {
+  project      = "${var.gcp_project_id}"
+  config       = "${var.spannerinstance["config"]}"
```

Comment:
You need to declare the `spannerdatabase` variable within this terraform file. Check around line 190 for similar declarations. Same for `spannerinstance`

Reply:
ya. forgot to declare. Strange, it didnt catch during my local test.